### DISCUSSION TITLE
Where have the asset metadata plots gone?

### DISCUSSION THREAD

**Discussion by the4thamigo-uk** [dagster-io/dagster#24574] [Q&A]
I seem to remember being able to view the metadata plots from the asset details page? Now, I can only find them via the Global Asset Lineage page. This is kinda annoying... Is there a plan to bring it back?

**Comment by garethbrickman** [comment]
I can see the metadata plot in the UI after materializing the asset. This is using dagster 1.8.7: 
![Screenshot 2024-09-18 at 16 24 46](https://github.com/user-attachments/assets/323e1c95-e35e-4fb7-bfdb-2f3664af13fa)

**Reply by the4thamigo-uk** [reply]
Its weird, it says no metadata is found, but yet I definitely have metadata, and the plot shows up in the asset graph.

**Reply by the4thamigo-uk** [reply]
Im using dagster v1.8.5, have there been bug fixes in this area?

**Reply by the4thamigo-uk** [reply]
I also see the issue in v1.8.6, but I have dbt issues with 1.8.7 so I cant try that easily

**Reply by the4thamigo-uk** [reply]
![image](https://github.com/user-attachments/assets/df57ab8d-81ce-4c8f-aec5-6b545013b734)

Clicking on the 'View in graph' gives the metadata plot : 

![image](https://github.com/user-attachments/assets/4c41e091-a7cd-4522-868c-78a6012cac50)

When I go the the partitions tab I can see the meta data : 

![image](https://github.com/user-attachments/assets/cff66202-b0b7-4cc5-bb96-5f290fdd6edc)

**Reply by the4thamigo-uk** [reply]
@garethbrickman Ah it seems that it only works when the latest partition is materialized

I think this may be related to the graphql query snippet here : 

```
query AssetOverviewMetadataEventsQuery($assetKey: AssetKeyInput!) {
  assetOrError(assetKey: $assetKey) {
    ... on Asset {
      id
      assetMaterializations(limit: 1, partitionInLast: 1) {
        timestamp
        runId
        metadataEntries {
          ...MetadataEntryFragment
          __typename
        }
        __typename
      }
      assetObservations(limit: 1, partitionInLast: 1) {
        timestamp
        runId
        metadataEntries {
          ...MetadataEntryFragment
          __typename
        }
        __typename
      }
      __typename
    }
    __typename
  }
}
```

**Reply by the4thamigo-uk** [reply]
https://github.com/dagster-io/dagster/issues/24600

**Comment by the4thamigo-uk** [comment]
https://github.com/dagster-io/dagster/issues/24600
